WORKSPACE

```diff
@@ -122,3 +124,28 @@ maven_install(
 load("@maven//:compat.bzl", "compat_repositories")
 
 compat_repositories()
+
+
+rules_scala_version = "20220201"
```

Comment:
Rename to `RULES_SCALA_VERSION`.

Conventionally in Starlark, global variable names are all caps.